### PR TITLE
upgrade to cats 1.0.0-MF

### PR DESCRIPTION
**Pull request by matsluni**
This PR upgrades cats to 1.0.0-MF.

### CODE REVIEW

File: util/dependencies.sbt [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
 libraryDependencies ++= Seq(
-  "com.typesafe.scala-logging" %% "scala-logging" % "3.7.2",
-  "joda-time" % "joda-time" % "2.9.9",
-  "org.joda" % "joda-convert" % "1.8.2",
-  "org.typelevel" %% "cats-macros" % catsVersion,
-  "org.typelevel" %% "cats-core" % catsVersion,
-  "org.json4s" %% "json4s-scalap" % "3.5.2"
+  "com.typesafe.scala-logging"  %% "scala-logging"  % "3.7.2",
+  "joda-time"                   %  "joda-time"      % "2.9.9",
```
Comment (schleichardt) [inline util/dependencies.sbt]: this kind of formatting style produces a lot of noice in a upcoming PR when you add a lib with a longer name.

Comment (matsluni) [inline util/dependencies.sbt]: I like this formatting style more. I think it helps too read the dependencies more clearly. 
There is a Github url parameter `w=1` which ignores whitespace changes. I think this should help reading a PR with a lot of whitespace changes.

**Comment by matsluni** [comment]
What is the next step after merging this? Increase version and do a release?

**Comment by rjsvaljean** [comment]
I think you should do sbt release: https://github.com/sphereio/sphere-scala-libs/blob/master/HACKING.md

**Pull request merged**